kafka/testhelpers.go

```diff
@@ -125,3 +125,18 @@ func getMessageCountInTopic(topic string) (int, error) {
 
 	return cnt, nil
 }
+
+// getBrokerList returns a list of brokers (ids) in the cluster
+func getBrokerList(H Handle) (brokers []int32, err error) {
+	md, err := getMetadata(H, nil, true, 15*1000)
```

Comment:
this timeout is not a `Duration`... although arguably a small thing, people will use detail like this as a proxy for their opinion of the entire library - getting them right is very important (yep, i know this is an existing decision, but pointing it out anyway). And using an int for a timeout is a bad idea (though writing it as 15*1000 is a great idea).

Reply:
Contexts did not exist when getMetadata() was added.

Reply:
`Duration` did though...